### DISCUSSION TITLE
mail onayını kapatma

### DISCUSSION THREAD

**Discussion by gulasor** [realsuayip/django-sozluk#102] [Q&A]
Hocam selam. mail onayını alma özelliğini nasıl kapatabilirim? Ayrıca database alanında bilgi sahibi biri değilim hatta zerre bir şey bilmiyorum. Docker ile localhost olarak kuruyor sanırsam. Acaba yazılımı başka bir sunucuya verileri kaybetmeden nasıl taşıyabilirim?

**Comment by realsuayip** [comment]
> mail onayını alma özelliğini nasıl kapatabilirim

bir python developer ile görüşüp yaptırmanız gerek, benim tarifimle yapabileceğiniz bir şey değil. ben şahsen şu an böyle bir servis  (ek geliştirme, özellik değiştirme) vermiyorum.

> yazılımı başka bir sunucuya verileri kaybetmeden nasıl taşıyabilirim

yine benzer bir şekilde.

> Docker ile localhost olarak kuruyor sanırsam

sunucunuza da docker kullanarak kurabilirsiniz.
